target_cratedb/connector.py

```diff
@@ -5,25 +5,10 @@
 from builtins import issubclass
 from datetime import datetime
 
-import sqlalchemy
 import sqlalchemy as sa
 from crate.client.sqlalchemy.types import ObjectType, ObjectTypeImpl, _ObjectArray
```

Comment:
That we need to import both `ObjectType` and `ObjectTypeImpl` here, and use them correspondingly, clearly indicates something is not optimal, and should be addressed on behalf of improvements to the type definitions of the CrateDB SQLAlchemy dialect in [crate-python](https://github.com/crate/crate-python).